packages/component-library/src/components/form/mt-number-field/mt-number-field.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { mount } from "@vue/test-utils";
```

Comment:
Instead of creating a new Test file, please write a story for this: 
https://github.com/shopware/meteor/blob/main/packages/component-library/src/components/form/mt-number-field/mt-number-field.interactive.stories.ts

Reply:
done